tests/testthat/test-geom-dotplot.R

```diff
@@ -125,6 +125,34 @@ test_that("geom_dotplot draws correctly", {
     ggplot(dat, aes(x)) + geom_dotplot(binwidth = .4, stackdir = "centerwhole") + coord_flip()
   )
 
+  # Stacking methods with stackratio < 1
+  expect_doppelganger("stack up, stackratio = 0.5",
+    ggplot(dat, aes(x)) + geom_dotplot(binwidth = .4, stackdir = "up", stackratio = 0.5)
+  )
+  expect_doppelganger("stack down, stackratio = 0.5",
```

Comment:
I think one test with non-default stack ratio would be adequate. Visual tests are expensive to maintain so we really want to keep them as minimal as possible.

Reply:
Done.